### PR TITLE
[TOMEE-2742] Add Spanish translation for simple-jms

### PR DESCRIPTION
**Pull request by emecas**

### CODE REVIEW

**Comment by jgallimore** [comment]
Could we get a Spanish speaker to review? @cesarhernandezgt ?

File: examples/simple-jms/README_es.adoc [hunk 1]
```diff
@@ -0,0 +1,253 @@
+:index-group: JMS and MDBs
+:jbake-type: page
+:jbake-status: status=published
+
+= Simple JMS
+
+Este ejemplo demuestra cómo configurar un servicio JMS personalizado ``CustomJmsService`` 
+para producir y consumir un Mensaje JMS ``Message``.
+
+= Codigo
```
Comment (cesarhernandezgt) [inline examples/simple-jms/README_es.adoc]: Código

File: examples/simple-jms/README_es.adoc [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+== El servicio JMS: ``Message``, ``Queue``, MessageProducer``, ``MessageConsumer``
+
+Aquí tenemos un punto final REST, una clase con la anotacion: ``@Path("/ message")`` que indica la ruta correspondiente a la clase ``CustomJmsService``. Por lo tanto, definimos ``sendMessage()`` como ``@POST`` y ``acceptMessage()`` como ``@GET`` para la  ruta ``/message``.
```
Comment (cesarhernandezgt) [inline examples/simple-jms/README_es.adoc]: anotación

File: examples/simple-jms/README_es.adoc [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+= Running the app
+
+Ejecutar el ejemplo es simple. En el directorio ``simple-jms`` ingrese:
+
+....
+$ mvn tomee:run
+....
+
+Esto creara una salidad similiar a lo siguiente.
```
Comment (cesarhernandezgt) [inline examples/simple-jms/README_es.adoc]: salida similar

**Pull request merged**